src/Serilog.Sinks.Email/Sinks/Email/Email.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Serilog.Sinks.Email.Sinks.Email
+{
+    internal class Email
+    {
+        public string From { get; set; }
+
+        public string Subject { get; set; }
+
+        public string Body { get; set; }
+
+        public bool IsBodyHtml { get; set; }
+        public IEnumerable<string> Tos { get; set; }
```

Comment:
Nit: whitespace

Nit: could just be `To` (the plural would be implied by the collection value)

Would it be possible to make this class immutable, by accepting all of these values into the constructor, and making the properties get-only?